core/bind/core_bind.cpp

```diff
@@ -787,7 +787,7 @@ void _OS::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "exit_code"), "set_exit_code", "get_exit_code");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "low_processor_usage_mode"), "set_low_processor_usage_mode", "is_in_low_processor_usage_mode");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "low_processor_usage_mode_sleep_usec"), "set_low_processor_usage_mode_sleep_usec", "get_low_processor_usage_mode_sleep_usec");
-	ADD_PROPERTY(PropertyInfo(Variant::STRING, "tablet_driver"), "set_current_tablet_driver", "get_current_tablet_driver");
+	ADD_PROPERTY(PropertyInfo(Variant::STRING, "tablet_driver", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_IGNORE_VALUE_IN_DOCS), "set_current_tablet_driver", "get_current_tablet_driver");
```

Comment:
There's `ADD_PROPERTY_DEFAULT()` that can be used to override the default value for the docs already, so maybe we can use that instead of adding another usage hint? That would reduce the boilerplate somewhat (we'd lose the `platform-dependent` default value though, which would be replaced by a sane null value - here likely `""`).

Reply:
> There's `ADD_PROPERTY_DEFAULT()`

Seems reasonable, I'll test how it works and update PR.

> we'd lose the `platform-dependent`

No we won't, it's for constants where some value is always expected, properties simply do not display any default value if it's equal to `""` (it was this way before and not changed in this PR).

Reply:
`ADD_PROPERTY_DEFAULT` works OK, I have removed PROPERTY_USAGE change.